demisto_sdk/commands/test_content/test_modeling_rule/test_modeling_rule.py

```diff
@@ -311,44 +339,59 @@ def verify_results_against_test_data(
             )
         table_result = create_table(expected_values, result)
         logger.info(f"\n{table_result}")
-        for key, val in expected_values.items():
-            if val:
-                result_val = result.get(key)
-                out = f"Checking for key {key}:\n - expected: {val}\n - received: {result_val}"
+        for expected_key, expected_value in expected_values.items():
+            if expected_value:
+                received_value_before = result.get(expected_key)
```

Comment:
Rename:
* received_value_before -> received_value
* received_value (in the tuple result of the call to transform_received_value_by_expected_type) -> received_value_sanitized
* received_value_type -> received_value_sanitized_type